### PR TITLE
Update @redhat-cloud-services packages

### PR DESCRIPTION
**Pull request by dlabrecq**
Updated @redhat-cloud-services packages to latest

https://issues.redhat.com/browse/COST-909

### CODE REVIEW

File: src/api/api.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { authInterceptor as insightsAuthInterceptor } from '@redhat-cloud-services/frontend-components-utilities/files/interceptors';
+import { authInterceptor as insightsAuthInterceptor } from '@redhat-cloud-services/frontend-components-utilities/esm/interceptors';
```
Comment (boaz0) [inline src/api/api.ts:1]: It is supposed to be `@redhat-cloud-services/frontend-components-utilities/interceptors`
See here: https://github.com/RedHatInsights/frontend-components/tree/master/packages/utils#import-path-change

**Review by boaz0 — COMMENTED** [review]
You need to update the imports according to the docs otherwise the purpose of the upgrade is missing.

File: src/api/api.ts (hunk 1 above)
Comment (dlabrecq) [reply src/api/api.ts]: Thanks, updated

**Review by ddonahue007 — APPROVED** [review]
LGTM

**Comment by dlabrecq** [comment]
Thanks!

**Pull request merged**